core/src/main/java/jenkins/security/apitoken/ApiTokenStore.java

```diff
@@ -371,6 +371,22 @@ private void init() {
             
             return result;
         }
+    
+        public static @Nonnull HashedToken buildNewFromLegacy(@Nonnull HashValue value, boolean migrationFromExistingLegacy) {
```

Comment:
`@since` or `@Restricted`

Reply:
as the parent class (ApiTokenStore) is `@Restricted`, this method is also `@Restricted` by implication